tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/views/inactive/BUILD

```diff
@@ -4,6 +4,22 @@ package(default_visibility = ["//tensorboard:internal"])
 
 licenses(["notice"])  # Apache 2.0
 
+py_binary(
+    name = "inline_images",
+    srcs = ["inline_images.py"],
+    python_version = "PY3",
+    srcs_version = "PY2AND3",
+    deps = [],
```

Comment:
I forget... Is `deps` field required? If not, just drop it?

Reply:
It is optional if empty (see https://docs.bazel.build/versions/master/be/python.html). Dropped it.